addon/components/animated-container.ts

```diff
@@ -72,7 +71,7 @@ export default class AnimatedContainerComponent extends Component {
     @argument tag
     @type String
   */
-  tag = 'div';
+  tag = this.args.tag || 'div';
```

Comment:
@jacojoubert component extends from `@ember/component`, so it won't have `this.args` I believe?

I think to make it work, you need to make a change in `addon/templates/components/animated-container.hbs` on line 4 like so:
```hbs
{{#let (element (or @tag "div")) as |Tag|~}}
```